CMakeLists.txt

```
@@ -22,4 +22,4 @@ target_compile_features(geo-test PUBLIC cxx_std_17)
 if (NOT MSVC)
   set_target_properties(geo-test PROPERTIES COMPILE_FLAGS "-Wall -Wextra")
 endif()
-target_link_libraries(geo-test geo doctest)
+target_link_libraries(geo-test geo utl doctest)
```

```suggestion
target_link_libraries(geo-test geo doctest)
```

Der Test selbst bindet kein `utl` ein.

---

Applied in 1fbf9d089e76a58a86cc6c76dff9d08618de40f5